routes/htmlRoutes.js

```diff
@@ -3,59 +3,58 @@ var db = require("../models");
 // Requiring our custom middleware for checking if a user is logged in
 var isAuthenticated = require("../config/middleware/isAuthenticated");
 
-module.exports = function (app) {
+module.exports = function(app) {
   // Load index page
-  app.get("/", function (req, res) {
+  app.get("/", function(req, res) {
     if (req.user) {
       res.redirect("/members");
     } else {
-      res.render("index")
-    };
+      res.render("index");
+    }
   });
 
-
-  app.get("/account", function (req, res) {
+  app.get("/account", function(req, res) {
     if (req.user) {
       res.render("account", {
         email: req.user.email,
         id: req.user.id,
         createdAt: req.user.createdAt.substring(0, 10)
-      })
+      });
     } else if (!req.user) {
```

Comment:
I forgot to update this part to just say `else` instead of `else if (!req.user)`

Reply:
And done.